### PR TITLE
fix a few bugs

### PR DESCRIPTION
**Pull request by plally**

### CODE REVIEW

File: lua/cfcfactions/core/sv/sv_player_ext.lua [hunk 1]
```diff
@@ -22,19 +22,15 @@ end
...
 --Obtains the player's faction id, or 0 if not. Bots always return 'b0t'
 function meta:GetFactionID()
     if self:IsBot() then return "b0t" end
     if self:IsPlayer() then
-        if self:IsInFaction() == false then
-            return nil
-        else 
-            return fpm[self:SteamID64()].FactionID
-        end
+       return fpm[self:SteamID64()] and fpm[self:SteamID64()].FactionID
```
Comment (overlordbleck) [inline lua/cfcfactions/core/sv/sv_player_ext.lua:33]: Wouldn't this just return false...?

File: lua/cfcfactions/core/sv/sv_player_ext.lua [hunk 2]
```diff
@@ -22,19 +22,15 @@ end
 function meta:IsInFaction()
     if self:IsMerc() == true then return true end
     
-    local userHasFaction = self:GetFactionID() ~= 0
+    local userHasFaction = self:GetFactionID() ~= nil
```
Comment (overlordbleck) [inline lua/cfcfactions/core/sv/sv_player_ext.lua:25]: #NilSquad

File: lua/cfcfactions/core/sv/sv_player_ext.lua (hunk 1 above)
Comment (plally) [reply lua/cfcfactions/core/sv/sv_player_ext.lua:33]: no, the way "and" works is if the first thing is true it returns the second thing.
else it just returns the first thing
{} and "test" returns "test"
nil and "test" returns nil

Comment (overlordbleck) [reply lua/cfcfactions/core/sv/sv_player_ext.lua:33]: Huh, didn't know that, quite the confusing syntax

File: lua/cfcfactions/core/sv/sv_init.lua [hunk 3]
```diff
@@ -68,7 +68,7 @@ hook.Add("Initialize", "cfcInitializeFactions", cfcFactions:InitializeFactions()
 --Player Say Hook
 --Handles if a player wishes to open the faction menu by typing the command
 local function cfcPlayerSay(ply, msg)
-    if string.len(cfcFactions.Config.CHAT_COMMAND) > 0 then
+    if string.len(cfcFactions.Config.CHAT_COMMAND or "") > 0 then
```
Comment (justinmuskopf) [inline lua/cfcfactions/core/sv/sv_init.lua:71]: Is this protection against a case where `CHAT_COMMAND` doesn't exist?

Comment (plally) [reply lua/cfcfactions/core/sv/sv_init.lua:71]: yes

**Review by sarahsturgeon — APPROVED** [review]
Clean. Good job.

**Pull request merged**